.github/CODEOWNERS

```diff
@@ -31,7 +31,7 @@
 # ServiceOwners:                                                   @miaojiang
 
 # PRLabel: %App Configuration
-/sdk/data/azappconfig/                                             @jhendrixMSFT @rickwinter
+/sdk/data/azappconfig/                                             @jhendrixMSFT @rickwinter @RichardChen820 @linglingye001
```

Comment:
Can you please change the ordering here. The convention is that the left-most alias is the main point-of-contact, and the rest are backups. e.g. `[primary owner, backup1, backup2, ...backupN]`. You can put me last and remove Rick.